packages/@tailwindcss-browser/tsup.config.ts

```diff
@@ -13,4 +13,20 @@ export default defineConfig({
     'process.env.NODE_ENV': '"production"',
     'process.env.FEATURES_ENV': '"stable"',
   },
+  esbuildPlugins: [
+    {
+      name: 'patch-intellisense-apis',
+      setup(build) {
+        build.onLoad({ filter: /intellisense.ts$/ }, () => {
+          return {
+            contents: `
+              export function getClassList() { return [] }
```

Comment:
Does this mean these APIs aren't available when running the browser version?

Reply:
These APIs are internal and were already inaccessible in the `@tailwindcss/browser` package. They were on an (i.e. not tree-shakable) which meant they still took up some space. This replaces those APIs so they take up much less space since they can't be used anyway.

basically this is just a code-size reduction and doesn't eliminate any actual functionality.

Reply:
Ah that must be it great, thanks mate